Lib/test/test_syntax.py

```diff
@@ -978,10 +975,11 @@ def func2():
 """
         self._check_error(code, "invalid syntax")
 
+    @unittest.skipIf(support.use_old_parser(), "The old parser ")
```

Comment:
Why add both `skipIf` and also change the test itself?

Reply:
This is probably a copypasta on Pablo's part as evidenced by the unfinished message. Let me remove it and see if tests still pass due to the change in the test body.
```suggestion
```